test

```diff
@@ -57,7 +57,16 @@ function unit_tests {
 }
 
 function integration_tests {
-	echo "Running integration tests..."
+	MIGRATION_TEST="v3.0.2"
+	if [ -n "$MIGRATION_TEST_VERSION" ]; then
+		MIGRATION_TEST=$MIGRATION_TEST_VERSION
+	fi
+	echo "Running integration, migration tests with" etcd $MIGRATION_TEST
+	curl -L https://github.com/coreos/etcd/releases/download/$MIGRATION_TEST/etcd-$MIGRATION_TEST-linux-amd64.tar.gz -o /tmp/etcd-$MIGRATION_TEST-linux-amd64.tar.gz
```

Comment:
I think this test should be optional if it has to pull down binaries from github.